enterprise_subsidy/apps/transaction/management/commands/write_reversals_from_enterprise_unenrollments.py

```diff
@@ -73,13 +81,17 @@ def unenrollment_can_be_refunded(
         # https://2u-internal.atlassian.net/browse/ENT-6825
         # OCM course refundability is defined as True IFF:
         # ie MAX(enterprise enrollment created at, course start date) + 14 days > unenrolled_at date
-        enrollment_created_at = enterprise_course_enrollment.get("created")
-        enrollment_unenrolled_at = enterprise_course_enrollment.get("unenrolled_at")
+        enrollment_created_at = self.strip_microseconds_from_datetime_str(enterprise_course_enrollment.get("created"))
+
+        enrollment_unenrolled_at = self.strip_microseconds_from_datetime_str(
+            enterprise_course_enrollment.get("unenrolled_at")
+        )
+
         enrollment_created_datetime = datetime.strptime(
             enrollment_created_at, "%Y-%m-%dT%H:%M:%SZ"
         )
         course_start_datetime = datetime.strptime(
-            course_start_date, "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
nit: you could just add microseconds into the string pattern :) 
https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Reply:
What's tricky is that I don't know if all the datetime strings have microseconds, so I'm not sure how to account for both formats

Reply:
nvm forgot about try catches 🤪